.github/workflows/release-please.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches:
+      - master
+name: release-please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v2
```

Comment:
We should add the `name:` release to this step, so that it's properly referenced by `steps.release`.

If I have this incorrectly documented anywhere, mind pointing me in the direction to fix it.

Reply:
So this was mostly my fault :) The first sample here has no id/name:
https://github.com/google-github-actions/release-please-action/blob/main/README.md#setting-up-this-action

Eventually I figured out I needed the full sample with npm publish, and then I messed up the copy of the additional yamls:
https://github.com/google-github-actions/release-please-action/blob/main/README.md#automating-publication-to-npm